src/directory/directory.mjs

```diff
@@ -841,7 +841,7 @@ export const directory = {
               path: 'src/pages/[platform]/reference/project-structure/index.mdx'
             },
             {
-              path: 'src/pages/[platform]/reference/amplifyconfiguration/index.mdx'
+              path: 'src/pages/[platform]/reference/amplify_outputs.json/index.mdx'
```

Comment:
Feels a little weird to put .json in the path, can we just call this amplify_outputs

Reply:
fair, it's been removed